test/cloudformation/iam_cloudformation.yaml

```diff
@@ -66,12 +66,15 @@ Resources:
               - iam:GetRole
               - iam:GetRolePolicy
               - iam:ListAttachedRolePolicies
+              - iam:ListOpenIDConnectProviders
+              - iam:TagOpenIDConnectProvider
```

Comment:
Can we add these policies to the list of OpenID permissions? I believe there's a block that already covers OIDC permissions right now